pom.xml

```diff
@@ -202,13 +188,19 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.46</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>mailer</artifactId>
-            <version>1.21</version>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- Upper bounds between Pipeline: Global Lib and HTML Unit -->
```

Comment:
Is this still an issue? I thought this was solved a few months ago.

Reply:
Fixed but unreleased, I have been meaning to get to it.